index.js

```diff
@@ -0,0 +1,30 @@
+const taskInput = document.getElementById('myInput');
+const taskList = document.getElementById('taskList');
+
+taskInput.addEventListener('keydown', function (event) {
+  const taskText = taskInput.value.trim();
+  if (event.key === 'Enter' && taskText !== '') {
+    const taskItem = createTaskItem(taskText);
+    taskList.appendChild(taskItem);
+    taskInput.value = '';
+  }
+});
+
+function createTaskItem(text) {
```

Comment:
해당 함수는 HTML Element를 만들고 있어서 함수명에 이 사실이 들어가도 좋을 것 같아요~

```suggestion
function createTaskItemElement(text) {
```

Reply:
넵 수정했습니다~